build.sbt

```diff
@@ -19,6 +21,7 @@ lazy val root = Project("frameless", file("." + "frameless")).in(file("."))
   .aggregate(core, cats, dataset, ml, docs)
   .settings(framelessSettings: _*)
   .settings(noPublishSettings: _*)
+  .settings(mimaPreviousArtifacts := Set(), mimaFailOnNoPrevious := false)
```

Comment:
To skip `mimaReportBinaryIssues` command in the `root` and `mdocs` sub projects.